modelscope/utils/plugins.py

```diff
@@ -263,14 +266,22 @@ def install_module_from_requirements(requirement_path, ):
 
     """
 
-    install_args = ['-r', requirement_path]
-    status_code, _, args = PluginsManager.pip_command(
-        'install',
-        install_args,
-    )
-    if status_code != 0:
-        raise ImportError(
-            f'Failed to install requirements from {requirement_path}')
+    install_list = []
+    with open(requirement_path, 'r', encoding='utf-8') as f:
+        requirements = f.read().splitlines()
+        for req in requirements:
+            installed, _ = PluginsManager.check_plugin_installed(req)
```

Comment:
If a package is installed, but user requires a newer version of this package, if seems that it will skip install the newer version

Reply:
The PluginsManager.check_plugin_installed will valid two things:
1 . if this package is installed.
2. if 1 is yes, if the string of package with a version spec, this method will verify whether the installed version contains the provided version.  If it contains the provided version, then it return true, otherwise false.